src/utils/worker/resolve.ts

```diff
@@ -2,4 +2,7 @@ import * as emailTemplates from './mail/templates';
 import { sendMail } from './mail';
 import { JobDataMap } from './create';
 
-export const email = ({ template, data }: JobDataMap['email']) => sendMail(emailTemplates[template](data));
+export const email = async ({ template, data }: JobDataMap['email']) => {
+    const renderedTemplate = await emailTemplates[template](data);
```

Comment:
I think I can rollback it and remove async from template functions

Reply:
It probably doesn't matter since it's a background job anyway